templates/shortcode-form-grid.php

```diff
@@ -364,10 +365,8 @@ static function($term)use($tag_text_color,$tag_bg_color){
                                             );
                                             echo sprintf(
                                             /* translators: 1: amount of income raised 2: goal target amount. */
-                                                __( '<div class="form-grid-raised__details">
-                                                   <span class="amount"  data-amounts="%1$s">%2$s</span>
-                                                   <span class="goal" data-amounts="%3$s"><span>of </span>%4$s</span>
-                                         </div>', 'give' ),
+                                                __( '<span class="amount"  data-amounts="%1$s">%2$s</span>
+                                                     <span class="goal" data-amounts="%3$s"><span>of </span>%4$s</span>', 'give' ),
```

Comment:
Should these even be translatable? Goal target amount is a number, no?

Reply:
@kjohnson Yea it's a number.  The word "of" is in the span. I also remove the surrounding span in a later commit

Reply:
Ah, I missed that part. Ok.